test/extended/two_node/utils/common.go

```diff
@@ -317,3 +777,95 @@ func isNodeObjReady(node corev1.Node) bool {
 	}
 	return false
 }
+
+// isPodReady checks if a pod is ready based on its conditions
+func isPodReady(pod *corev1.Pod) bool {
```

Comment:
Lets remove this and use `podutils.IsPodReady(pod)` from the `"k8s.io/kubectl/pkg/util/podutils"` package